boards/riscv/longan_nano/longan_nano_lite.dts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
what is the purpose of this file? seems to be a duplicate

Reply:
Longan Nano is available two variant.
Longan Nano is implementing gd32vf103cbt6 (128kb flash/ 32kb sram) SoC.
Longan Nano Lite is implementing gd32vf103c8t6 (64kb flash/ 20kb sram) SoC.

Lite seems that only a small amount is in circulation and it is not listed on Sipeed's website.
I create the settings based on the platformio's configuration
https://docs.platformio.org/en/latest/boards/gd32v/sipeed-longan-nano-lite.html.